webapp/src/views/SignUp.vue

```diff
@@ -100,21 +100,33 @@
                         class="uppercase"
                         ref="captchaField"
                         tabindex="4"
+                        hint="Can\'t see? Hear an audio CAPTCHA instead."
```

Comment:
You could make this depend on `kind`, like `:hint="kind == 'image' ? 'Can\'t see? Hear an audio CAPTCHA instead.' : 'Trouble hearing? Switch to an image CAPTCHA.'"`

Reply:
done